cristin-import/src/test/resources/features/BookFeatures.feature

```diff
@@ -0,0 +1,102 @@
+Feature: Book conversion features
```

Comment:
All rules from the BookAnthology and the BookMonograph were moved here. Rules that applied to both types were adapted in order for both types to be checked

Reply:
Should this file be called BookRules to match the rest of the feature files?